requirements-dev.txt

```diff
@@ -0,0 +1,17 @@
+-r requirements.txt
+black==18.9b0
+codecov==2.0.15
```

Comment:
Not needed generally

Reply:
pinned to help renovate create explicit PRs to help us notice if changes in our coverage metrics stem from codecov bumps.